content/ja/examples/service/networking/default-ingressclass.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.k8s.io/v1
+kind: IngressClass
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+  name: nginx-example
+  annotations:
+    ingressclass.kubernetes.io/is-default-class: "true"
+spec:
+  controller: k8s.io/ingress-nginx
```

Comment:
please add newline

Reply:
Thank you for your kind instructions! I've fixed this error.